diff --git a/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js b/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
index 2531bf9..e2c155c 100644
--- a/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
+++ b/node_modules/gatsby/dist/utils/parcel/compile-gatsby-files.js
@@ -37,7 +37,36 @@ function constructParcel(siteRoot) {
     targets: {
       root: {
         outputFormat: `commonjs`,
-        includeNodeModules: false,
+        includeNodeModules: [
+          "decode-named-character-reference",
+          "micromark",
+          "micromark-core-commonmark",
+          "micromark-extension-footnote",
+          "micromark-extension-gfm",
+          "micromark-extension-gfm-autolink-literal",
+          "micromark-extension-gfm-strikethrough",
+          "micromark-extension-gfm-table",
+          "micromark-extension-gfm-tagfilter",
+          "micromark-extension-gfm-task-list-item",
+          "micromark-factory-destination",
+          "micromark-factory-label",
+          "micromark-factory-space",
+          "micromark-factory-title",
+          "micromark-factory-whitespace",
+          "micromark-util-character",
+          "micromark-util-chunked",
+          "micromark-util-classify-character",
+          "micromark-util-combine-extensions",
+          "micromark-util-decode-numeric-character-reference",
+          "micromark-util-encode",
+          "micromark-util-html-tag-name",
+          "micromark-util-normalize-identifier",
+          "micromark-util-resolve-all",
+          "micromark-util-sanitize-uri",
+          "micromark-util-subtokenize",
+          "micromark-util-symbol",
+          "micromark-util-types",
+        ],
         sourceMap: false,
         engines: {
           node: `>= 14.15.0`
